barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/arithmetization.hpp

```diff
@@ -119,6 +119,7 @@ template <typename FF_> class UltraArith {
   public:
     static constexpr size_t NUM_WIRES = 4;
     static constexpr size_t NUM_SELECTORS = 11;
+    static constexpr size_t FIXED_BLOCK_SIZE = 1 << 10; // Size of each block in a structured trace
```

Comment:
Can you explain why this is the number we’ve chosen?

Reply:
Added a note that its currently chosen arbitrarily